### ISSUE TITLE
No module named 'fused'

### ISSUE THREAD

**Issue by Dong09** [wpeebles/gangealing#21]
hello,i have something problem!   wuuuu

i run this command : 
python -m torch.distributed.launch --nproc_per_node=8 train.py --ckpt cat --load_G_only --padding_mode border --vis_every 5000 --ckpt_every 50000 --iter 1500000 --tv_weight 1000 --loss_fn vgg_ssl --exp-name lsun_cats

![1](https://user-images.githubusercontent.com/45942487/170229491-d8d3accf-7137-45da-8391-ab8aaebbb230.png)
![2](https://user-images.githubusercontent.com/45942487/170229513-015c5be5-f493-419e-bc2e-afe8b3b980a2.png)
![3](https://user-images.githubusercontent.com/45942487/170229535-7a11fd77-3ac6-4a4f-a03c-e68e5ec01a5a.png)
![QQ截图20220525172209](https://user-images.githubusercontent.com/45942487/170229550-fb9e2a29-586c-4807-9c1c-f8059b4b9ca1.png)

thanks！

**Comment by wpeebles** [comment]
Hi @Dong09. Hmm, based on your error messages I guess the problem is likely that the StyleGAN2 extensions are failing to build, although I guess it's also possible that it's some relative path problem. Can you make sure the `gangealing` folder is added to your `PYTHONPATH`, and try re-running?

If that doesn't fix the problem, I'd definitely try running with a new version of PyTorch (the repo needs 1.10.1 or newer, and others have had issues compiling the StyleGAN extensions with old versions of PyTorch) and a recent version of CUDA (10+). If you still get errors I suppose it's possible it could be some Windows-specific issue. It seems quite likely to me that the GCC warning message in your first screenshot is related since it's commonly used for building PyTorch extensions. Maybe try following the link in the error message to upgrade GCC.

**Comment by Dong09** [comment]
Thanks!
Uninstall pytorch and install a new pytorch version of 1.10.1 has solved the problem.
And i have another question about training:
![QQ截图20220530140301](https://user-images.githubusercontent.com/45942487/170927105-0acc6186-540b-4ddf-875c-6a469d035527.png)
i run this command and then it always keep that.is it normal?

**Comment by wpeebles** [comment]
That warning message is from PyTorch distributed. I think it should be harmless. If you're training on 1 GPU, you can use `python` instead of `torchrun` and it'll probably go away. After you start training, you should see a progress bar pop-up after all of the networks are instantiated.

By the way, unrelated to this issue, but when training on 1 GPU you will likely get better results when using gradient accumulation to get a larger effective batch size (see issue https://github.com/wpeebles/gangealing/issues/10). The `--batch` argument controls the per-GPU batch size. Ideally, total batch size should be around 32 or 40, but anything greater than 5 should help improve results. If you end up using small batch sizes, using `--loss_fn lpips --tv_weight 2500` may be more stable (although to get high quality results you'll still need to use a batch size of ~32).

**Comment by Dong09** [comment]
> That warning message is from PyTorch distributed. I think it should be harmless. If you're training on 1 GPU, you can use `python` instead of `torchrun` and it'll probably go away. After you start training, you should see a progress bar pop-up after all of the networks are instantiated.
> 
> By the way, unrelated to this issue, but when training on 1 GPU you will likely get better results when using gradient accumulation to get a larger effective batch size (see issue #10). The `--batch` argument controls the per-GPU batch size. Ideally, total batch size should be around 32 or 40, but anything greater than 5 should help improve results. If you end up using small batch sizes, using `--loss_fn lpips --tv_weight 2500` may be more stable (although to get high quality results you'll still need to use a batch size of ~32).

This is very helpful to me and forgive me for not replying in time.
